source/ui/BioXAS/BioXASSSRLMonochromatorConfigurationView.h

```diff
@@ -0,0 +1,116 @@
+#ifndef BIOXASSSRLMONOCHROMATORCONFIGURATIONVIEW_H
+#define BIOXASSSRLMONOCHROMATORCONFIGURATIONVIEW_H
+
+#include <QGroupBox>
+#include <QDoubleSpinBox>
+#include <QComboBox>
+#include <QLayout>
+#include <QLabel>
+
+#include "beamline/BioXAS/BioXASSSRLMonochromator.h"
+#include "ui/BioXAS/BioXASSSRLMonochromatorRegionControlView.h"
+#include "ui/BioXAS/BioXASSSRLMonochromatorRegionControlEditor.h"
+
+#define BRAGG_OFFSET_MIN -1000
+#define BRAGG_OFFSET_MAX 1000
+#define BRAGG_SETTLING_TIME_MIN -1000
+#define BRAGG_SETTLING_TIME_MAX 1000
```

Comment:
Agreed.  This is a serious issue for REIXS.  The alignment and calibration of the spectrometer is hard coded magic numbers, which means that when I install the new gratings (when I eventually get the new gratings), and the calibration values change, not only will I have to tweak numerous hard-coded magic numbers, all previously collected data will not display correctly (the energy axis of old XES scans will be wrong).  

Creating a "beamlineCalibration" class that's a snap shot of all magic numbers is clearly overkill for this particular situation, but something like it may have some merit.